app/views/housing_rooms/show_building_rooms.html.erb

```diff
@@ -1,6 +1,9 @@
 <% content_for :header do %>
   <%= render 'components/page_header', :title => "#{@housing_building.name} Rooms" %>
 <% end %>
+<%= link_to "Floor Plans", floor_plans_path(:dorm_id => @housing_building.id), class: "button is-primary" %>
```

Comment:
this 'is-primary' button is not used on our website, perhaps we switch it to 'is-link is-info' for consistency